blog/config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+w17m5KEuEaOoK6SZhtwy7egprLGOLzll3UiSapqYvOvqIRJap1pMqhzHxUpMF9MvoF+Kh7IhaaBqSHseLNYscbbpR3zd+nScHILqVTcE30qt/EpI2p1C1a4w402hbbT1oemsBZ8euB7eOcdaPN4OSK+M0Cx0JfJv87F/ic4qvDI5fJ9tp2A+yaELGfuOAYO2zP59TDB1X6ILEu3eBC5wltjAQJmMg5rdl8srY+SvTJQa1I4Kw74tkKrwwv5JyDnbk0hmeGOHLeoPwcvt4/2IDEVh6rKieU81tnfKLL43m2nTBuyxMxLQ2cjFByXTqNN5GEGo1fNJCLmLkImyJhRzhTUdnBN7jVBqIJl52ytgAWB/qAbmxSaZ80xJ40QRMSPwk1u5LbvRZDH3zqBWjArpoAQ8OJWC0+hPl2oh--AhOyN83SEsiczQuE--UUFt9lO12kjTVQqK87AAFg==
```

Comment:
add an empty line at the end of every file where it is not added already.

Reply:
this is auto generated file